java/kafka/consumer/src/main/java/KafkaConsumerConfig.java

```diff
@@ -3,228 +3,56 @@
-            log.info("Configuring truststore");
-            props.put(CommonClientConfigs.SECURITY_PROTOCOL_CONFIG, "SSL");
-            props.put(SslConfigs.SSL_TRUSTSTORE_TYPE_CONFIG, "PEM");
-            props.put(SslConfigs.SSL_TRUSTSTORE_CERTIFICATES_CONFIG, config.getSslTruststoreCertificates());
-        }
-
-        if (config.getSslKeystoreCertificateChain() != null && config.getSslKeystoreKey() != null)   {
-            log.info("Configuring keystore");
-            props.put(CommonClientConfigs.SECURITY_PROTOCOL_CONFIG, "SSL");
-            props.put(SslConfigs.SSL_KEYSTORE_TYPE_CONFIG, "PEM");
-            props.put(SslConfigs.SSL_KEYSTORE_CERTIFICATE_CHAIN_CONFIG, config.getSslKeystoreCertificateChain());
-            props.put(SslConfigs.SSL_KEYSTORE_KEY_CONFIG, config.getSslKeystoreKey());
-        }
-
-        Properties additionalProps = new Properties();
-        if (!config.getAdditionalConfig().isEmpty()) {
-            StringTokenizer tok = new StringTokenizer(config.getAdditionalConfig(), System.lineSeparator());
-            while (tok.hasMoreTokens()) {
-                String record = tok.nextToken();
-                int endIndex = record.indexOf('=');
-                if (endIndex == -1) {
-                    throw new RuntimeException("Failed to parse Map from String");
-                }
-                String key = record.substring(0, endIndex);
-                String value = record.substring(endIndex + 1);
-                additionalProps.put(key.trim(), value.trim());
-            }
-        }
-
-        if ((config.getOauthAccessToken() != null)
-                || (config.getOauthTokenEndpointUri() != null && config.getOauthClientId() != null && config.getOauthRefreshToken() != null)
-                || (config.getOauthTokenEndpointUri() != null && config.getOauthClientId() != null && config.getOauthClientSecret() != null))    {
-            log.info("Configuring OAuth");
-            props.put(SaslConfigs.SASL_JAAS_CONFIG, "org.apache.kafka.common.security.oauthbearer.OAuthBearerLoginModule required;");
-            props.put(CommonClientConfigs.SECURITY_PROTOCOL_CONFIG, "SSL".equals(props.getProperty(CommonClientConfigs.SECURITY_PROTOCOL_CONFIG)) ? "SASL_SSL" : "SASL_PLAINTEXT");
-            props.put(SaslConfigs.SASL_MECHANISM, "OAUTHBEARER");
-            if (!(additionalProps.containsKey(SaslConfigs.SASL_MECHANISM) && additionalProps.getProperty(SaslConfigs.SASL_MECHANISM).equals("PLAIN"))) {
-                props.put(SaslConfigs.SASL_LOGIN_CALLBACK_HANDLER_CLASS, config.saslLoginCallbackClass);
-            }
-        }
-
-        // override properties with defined additional properties
-        props.putAll(additionalProps);
-
-        return props;
-    }
-
-    public String getBootstrapServers() {
-        return bootstrapServers;
+    public static String convertEnvVarToPropertyKey(String envVar) {
+        System.out.println("ENV_VAR is " + envVar);
```

Comment:
This should use logger if you want to log this (doesn't seem needed to me, but why not) or be removed if it is some leftover from debugging.

Reply:
Removed altogether